etc/answers-core.api.md

```diff
@@ -201,16 +201,19 @@ export interface FilterSearchResponse {
 }
 
 // @public
-export interface HighlightedValue {
-    fieldName: string;
+export interface HighlightedFieldLeaf {

```

Comment:
I'm thinking that I may actually prefer keeping it named HighlightedValue. Do you remember if we discussed any reasons why we shouldn't do that??

Reply:
don't think we discussed what to name this one, I can update to HighlightedValue